lib/src/Pos/Logic/Pure.hs

```diff
@@ -0,0 +1,266 @@
+-- | A pure logic layer. As such, it does nothing, but won't crash.
+
+module Pos.Logic.Pure
```

Comment:
I see no usages of this module. Is it supposed to be used in tests/benches?

Reply:
It was used in a simulation for initial benchmarking work, but that simulation isn't committed here. It's useful to have a pure logic layer though. It allows us to run a diffusion layer without any blockchain logic overhead.